l10n_be_mis_reports/data/new_mis_report_pl.xml

```diff
@@ -0,0 +1,398 @@
+<?xml version="1.0" ?>
+<!-- Copyright 2023 ACSONE SA/NV
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+  <record model="mis.report" id="new_mis_report_pl">
+    <field name="name">PROFIT AND LOSS ACCOUNT</field>
+    <field name="style_id" ref="mis_report_style_l10n_be_base" />
+  </record>
+  <record model="mis.report.kpi" id="mis_report_pl_acc_9900">
+    <field name="report_id" ref="mis_report_pl" />
+    <field name="name">acc_9900</field>
+    <field name="description">Gross margin [9900]</field>
+    <field name="expression">balp[9900%]</field>
+    <field name="type">num</field>
+    <field name="compare_method">pct</field>
+    <field name="style_id" ref="mis_report_style_l10n_be_2i" />
+    <field name="auto_expand_accounts" eval="True" />
+    <field
+            name="auto_expand_accounts_style_id"
+            ref="mis_report_style_l10n_be_acc_det"
+        />
+    <field name="sequence">10</field>
+  </record>
+  <record model="mis.report.kpi" id="mis_report_pl_acc_76A">
+    <field name="report_id" ref="mis_report_pl" />
+    <field name="name">acc_76A</field>
+    <field name="description">Of which: non-recurring operating income [76A]</field>
+    <field name="expression">balp[76A%]</field>
```

Comment:
@mathisjacoby This expression is suspicious. As far as I know, there is no account starting with `76A` in the official belgium chart of accounts. See for instance https://github.com/odoo/odoo/blob/16.0/addons/l10n_be/data/account.account.template.csv or https://plancomptablebelge.be/#classe-7